CHANGELOG.md

```diff
@@ -8,7 +8,22 @@
 
 **BREAKING CHANGES**
 
--   Rename `OlmMachine.init_from_store` introduced in v3.6.0 to `OlmMachine.initFromStore`.
+-   Rename `OlmMachine.init_from_store` introduced in v3.6.0 to
+    `OlmMachine.initFromStore`.
+    ([#84](https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/84))
+
+**BREAKING CHANGES**
+
```

Comment:
we now have two BREAKING CHANGES headings. (This is why I prefer tools like towncrier than maintaining changelog files in PRs, but I digress.)

```suggestion
```